README.md

```diff
@@ -0,0 +1,50 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
```

Comment:
Good job adding a professional and descriptive README for your repo! :clap: This would help recruiters or anyone who visits your repo to get a better understanding of what your project does :heavy_check_mark: